packages/web-twig/src/Resources/components/Button/Button.twig

```diff
@@ -18,7 +18,7 @@
 
 {# Attributes #}
 {%- set _disabledAttr = _isDisabled ? 'disabled' : null -%}
-{%- set _onClickAttr = _onClick ? 'onclick="' ~ _onClick ~ '"' : null -%}
+{%- set _onClickAttr = _onClick ? 'onclick=' ~ _onClick | escape('html_attr') : null -%}
```

Comment:
Why this? What effect does it have? When the user inserts js it is executed the same as with raw. Likewise in other cases

Reply:
I run a few manual tests through this, please see https://github.com/lmc-eu/spirit-design-system/pull/742#discussion_r1146807543.

But let's discuss this prop/attribute because it makes no sense to escape this, I think. We have discussed renaming it to `UNSAFE_onClick` but this makes no sense to me either. Since there could never exist any "safe" `onClick`. I am currently in doubt about what to do with this because it is not user input but the developer should care about the right script in `onClick`. And as this is only the proxy to the `onclick` attribute, maybe the best way will be to remove this and pass it via rest props, e.g. this `onClick` prop will be removed and you should use native `onclick` instead.

What do you think?